src/encoded/audit/experiment.py

```diff
@@ -1912,6 +2009,18 @@ def check_wgbs_lambda(bismark_metrics, threshold, pipeline_title):
                                    level='WARNING')
 
 
+def check_wgbs_lambda_ENCODE4(gembs_metrics, threshold, pipeline_title):
+    for metric in gembs_metrics:
+        conversion_rate = metric.get('conversion_rate')
+        if conversion_rate > threshold:
+            detail = (f'ENCODE experiment processed by {pipeline_title} '
+                f'pipeline has a lambda rate of {conversion_rate}. '
+                f'The lambda rate should be < 1.'
+            )
+            yield AuditFailure('high lambda C methylation ratio', detail,
```

Comment:
I think this message is a little confusing. I would propose something like this:
```suggestion
            yield AuditFailure('low lambda C conversion rate', detail,
```